src/stories/Modal/Modal.stories.tsx

```diff
@@ -49,6 +49,10 @@ const Template: Story<ModalProps & ComponentPropsWithoutRef<'div'> & { additiona
         setVisible((isVisible) => !isVisible);
     };
 
+    const handleOnBackdropClick = () => {
```

Comment:
Let's not add this unless you are going to add it as a separate story. The default behavior that we should demonstrate is the modal closing on backdrop click.

Reply:
Removed prop from stories